services/workflows-service/src/case-management/controllers/case-management.controller.ts

```diff
@@ -87,6 +87,16 @@ export class CaseManagementController {
           createdAt: true,
           firstName: true,
           lastName: true,
+          endUsersOnBusinesses: {
+            select: {
+              position: true,
+              business: {
+                select: {
+                  companyName: true,
+                },
+              },
+            },
+          },
           businesses: {
             select: {
               companyName: true,
```

Comment:
Is this still needed?

Reply:
I kept it as a backward compatibility measure in case we have other places that do not create a record in `EndUsersOnBusinesses`